string name
eager_core/State[] states
